### ISSUE TITLE
[Request] Add Support For Creating Hybrid Torrents

### ISSUE THREAD

**Issue by zero77** [anacrolix/torrent#445]
Can you please ad support for creating hybrid torrents as a default option but, with the option to create a v1 one torrent if required.

hybrid torrent test at the bottom:
https://blog.libtorrent.org/2020/09/bittorrent-v2/

**Comment by anacrolix** [comment]
See also https://github.com/anacrolix/torrent/issues/175.

**Comment by zero77** [comment]
@anacrolix 
Yes Bittorrent v2 support would be great but,  i imagine it will be a lot of work.

But, hybrid torrents are backward compatible and should work at the moment.
